### ISSUE TITLE
Improve efficiency of model

### ISSUE THREAD

**Issue by antoniogamiz** [antoniogamiz/tfg#54]
## Objetive

- [ ] Change the python interpreter to see if the model is faster (maybe `pypy` or similar)
- [ ] Reduce dataset by deleting not important tropes and films with really few tropes.
- [ ] Use softmax to compute the probability of the neural network output.
